app/mailers/activity_mailer.rb

```diff
@@ -23,7 +24,7 @@ def mail(options)
     template = options.delete(:template)
     if template
       prepend_view_path(File.dirname(template))
-      options[:template_path] = ''
+      options[:template_path] = File.dirname(template)
```

Comment:
I've removed `prepend_view_path` and now this mailer can only support relative path and there is no spec breaks.